features-json/asmjs.json

```diff
@@ -0,0 +1,241 @@
+{
+  "title":"asm.js",
+  "description":"an extraordinarily optimizable, low-level subset of JavaScript",
+  "spec":"http://asmjs.org/spec/latest/",
+  "status":"other",
+  "links":[
+    {
+      "url":"http://asmjs.org/",
+      "title":"Homepage"
+    },
+    {
+      "url":"https://github.com/dherman/asm.js/",
+      "title":"Source for spec and tools"
+    },
+    {
+      "url":"https://dev.modern.ie/platform/changelog/10532-pc/",
```

Comment:
This is probably a better link http://blogs.windows.com/msedgedev/2015/05/07/bringing-asm-js-to-chakra-microsoft-edge/